src/components/InputTodo.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
```

Comment:
![6HLE](https://user-images.githubusercontent.com/93735137/154529318-2125965e-23db-47e9-84a5-1a485bf74df1.gif)
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada tada tada